backend/dataall/core/environment/services/environment_service.py

```diff
@@ -213,6 +215,12 @@ def create_environment(uri, data=None):
         context = get_context()
         with context.db_engine.scoped_session() as session:
             EnvironmentRequestValidationService.validate_creation_params(data, uri, session)
+            if not OrganizationRepository.is_group_invited(session, uri, data['SamlGroupName']):
+                raise Exception(
+                    f'Group {data["SamlGroupName"]} is not a member of the organization {data.get("organizationUri")}. '
+                    f'Invite this group to the organisation before linking the environment.'
+                )
+
```

Comment:
Can we move to to part of `validate_user_groups()` in `EnvironmentRequestValidationService`

Reply:
done